frontend/web/components/ProjectFilter.tsx

```diff
@@ -30,7 +32,13 @@ const ProjectFilter: FC<ProjectFilterType> = ({
         (data || [])?.map((v) => ({ label: v.name, value: `${v.id}` })),
       )}
       onChange={(value: { value: string; label: string }) =>
-        onChange(value?.value || '')
+        onChange(
+          gitHubProjectFilterValidator
```

Comment:
Re-adding this comment, there's no reason to adjust this reusable component just for a GitHub integration. We can just provide {value} in GitHub's usage.

Reply:
I need this change to get not only the project id, because I need also the project name.

Reply:
Ok, then I think it'd be better to change onChange to 

```
onChange: (id: string, name:string) => void
```

Reply:
I made some changes. in the onChange, and remove gitHubProjectFilterValidator prop